src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -206,7 +208,43 @@ QString AccessibleItem::accessibleDescription() const
         return QString();
     }
 
-    return readable(m_element->accessibleExtraInfo());
+    Selection selection = m_element->score()->selection();
+
+    if (!selection.isRange() || !selection.startSegment() || !selection.endSegment()) {
+        return readable(m_element->accessibleExtraInfo());
+    }
+
+    String start = selection.startSegment()->formatBarsAndBeats();
+    String end;
+    if (Segment* endSeg = selection.endSegment()->prev1(SegmentType::ChordRest)) {
+        end = endSeg->formatBarsAndBeats();
+    }
+
+    start.remove(u';'); // Too many pauses in speech
+    end.remove(u';');
```

Comment:
This `remove()` can go inside the if statement above so it only happens when there is some end text.